staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -182,6 +182,10 @@ func (a *Authenticator) UpdateTransportConfig(c *transport.Config) error {
 		return &roundTripper{a, rt}
 	})
 
+	if c.HasTokenAuth() {
+		return nil
+	}
+
```

Comment:
Please add a comment explaining in detail (i.e. https://github.com/kubernetes/kubernetes/issues/87369#issuecomment-585579312) why we basically need the same check twice.

Reply:
Done.

Reply:
Done.